packages/Stylers/src/helpers.js

```diff
@@ -28,7 +28,29 @@ export const lineHeight = (scale = 0) => `
 export const spacer = numSpaces => `${Number.parseInt(tokens.space, 10) * numSpaces}px`;
 
 function zIsValid(level) {
-  const validLevels = ["auto", "initial", "inherit", "-1", "0", "1", "2", "3", "4", "5", "6", "7"];
+  const validLevels = [
+    "auto",
+    "initial",
+    "inherit",
+    "-1",
+    "0",
+    "1",
+    "2",
+    "3",
+    "4",
+    "5",
+    "6",
+    "7",
+    -1,
+    0,
+    1,
+    2,
+    3,
+    4,
+    5,
+    6,
+    7,
+  ];
```

Comment:
Maybe something like `/^[0-7]$|^-1$|inherit/g.test(level)` is shorter

Reply:
I like it!